tests/index.js

```diff
@@ -120,7 +122,13 @@ describe('Thorn', () => {
 
                         expect(request.url).to.contain(fixture.module);
                         expect(request.method).to.equal('POST');
-                        expect(request.data).to.eql(fixture.attributes);
```

Comment:
One of the things I noticed is that in this kind of tests where we're testing fixtures creation, we're not deeply comparing the values, `request.data`, being sent to the server which in turn might lead to the corner case we're fixing here, but on the other hand it also makes sense to do it this way since the data we're testing is actually the data we care about, whatever is being automagically filled by thorn/faker is considered accessory, which is also why I'm testing each individual data attribute on this test-case and not in the other ones.